api/src/aggregator/aggregator.controller.ts

```diff
@@ -81,6 +89,26 @@ export class AggregatorController {
       throw new HttpException(msg, HttpStatus.NOT_FOUND)
     }
 
+    // verify aggregator on load
```

Comment:
This verification would be nicer if we have in a separate function inside of service file.